src/robot/libraries/Collections.py

```diff
@@ -288,10 +302,9 @@ def reverse_list(self, list_):
         =>
         | ${L3} = ['c', 'b', 'a']
         """
-        self._validate_list(list_)
         list_.reverse()
 
-    def sort_list(self, list_):
+    def sort_list(self, list_: list):
```

Comment:
This could be `MutableSequence`. It needs a small adjustment to the implementation, though.